docs/terms-and-definitions.md

```diff
@@ -8,9 +8,66 @@ at the following addresses:
 - IEC Electropedia: available at
   [http://www.electropedia.org/](http://www.electropedia.org/)
 
+**computer hardware**
+
+All or part of the physical components of an information system. 
+(Leveraged from 'hardware' definition in ISO/IEC 2382:2015 Information technology -- Vocabulary)
+
+**computer program**
+
+Combination of computer instructions and data definitions 
+that enable computer hardware to perform computational or control functions. 
+(From ISO/IEC/IEEE 24765:2017 Systems and software engineering-Vocabulary)
+
+**firmware**
```

Comment:
The term "firmware" is not really used in the spec.

It appears in exactly one instance, in the `Software/SoftwarePurpose` vocabulary.

> - firmware: The Element provides low level control over a device's hardware.

But if we start defining all these terms, where will we stop? We also have software application, archive, configuration data, container image, device, device driver, executable, software library, platform, source, ...

Reply:
Discussed on tech call 4 Nov 2025:
- Agree the definitions need to align between the glossary and use within the spec
- Discussion on how many of the vocabulary terms need to be represented in the glossary
- Suggestion: If there is a reference to a specification in the Vocabulary and it is only used within a single profile, it doesn't need to be in the glossary